src/applications/disability-benefits/2346/config/form.js

```diff
@@ -58,7 +58,10 @@ const asyncReturn = (returnValue, error, delay = 300) =>
     }, delay);
   });
 
+const transformData = formData => JSON.stringify(formData);
+
 const submit = form => {
+  const submissionData = transformData(form.data);
```

Comment:
since you're only using this method once, there really isnt a need to extract it.

```
const submissionData = JSON.stringify(form.data)
```

Reply:
Got it!